web/src/App.js

```diff
@@ -91,6 +92,12 @@ class RootView extends React.Component {
                   <CLastChanges />
                 </Col>
               </Row>
+              <Row>
+                <Col>
+                  <CRepoChanges />
```

Comment:
I'm wondering to have a repository related events tab. Now we have Events stats, changes status, what about a Repo stats. Even if we will rework layout and where graph/stat should be, I think there is a room for that new tab:

- Top repository by merged changes
- Top repository by open changes
- Top 3 authors by repo
- Less active repositories
- ...

Reply:
yes good idea but in following PR

Reply:
yes in a following PR